lib/web/apiserver.go

```diff
@@ -762,6 +762,7 @@ func (h *Handler) bindDefaultEndpoints() {
 	// AWS OIDC Integration Actions
 	h.POST("/webapi/sites/:site/integrations/aws-oidc/:name/databases", h.WithClusterAuth(h.awsOIDCListDatabases))
 	h.POST("/webapi/sites/:site/integrations/aws-oidc/:name/deployservice", h.WithClusterAuth(h.awsOIDCDeployService))
+	h.GET("/webapi/sites/:site/integrations/:name/configure-deployservice-iam", h.WithLimiter(h.awsOIDCConfigureDeployServiceIAM))
```

Comment:
i think we should make the endpoint look similar to these `scripts` endpoint as https://github.com/gravitational/teleport/blob/81099f787c3df317b999de0c8d55752101340f35/lib/web/apiserver.go#L593

Reply:
Trying to adapt, it would be something like:
`/webapi/scripts/integration/configure/deployservice-iam.sh`
We can do that :+1:
We'll need to add the integration name as query param